content/en/docs/collector/transforming-telemetry.md

```diff
@@ -169,6 +169,43 @@ processors:
   k8sattributes/default:
 ```
 
+## Set status for span
+
+**Processor**:
+[span processor](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/spanprocessor)
+
+The
+[span processor](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/spanprocessor)
+can be used to change span status.
```

Comment:
```suggestion
To change the [status of a span](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/api.md#set-status), the
[span processor](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/spanprocessor) can be used.
```

Not a native speaker, but I had the feeling that the other way round it implies that this is all the span processor those, so by putting the thing you want to accomplish (change the span status) first, it's clearer what this is about.

Reply:
Let's get rid of passive voice altogether as per our style recommendations:

```suggestion
You can use the
[span processor](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/spanprocessor)
to change span status.
```